src/components/routes-positions.component.tsx

```diff
@@ -185,3 +189,62 @@ function TransportMarker(props: TransportMarkerProps) {
 function fromNowMinutes(date: Date): number {
   return Math.floor((new Date().getTime() - date.getTime()) / (60 * 1000))
 }
+
+const trolleyParks = [
+  {
+    name: 'Buiucani',
+    polygon: [
+      [47.03782046385607, 28.81591558456421],
+      [47.035422117402916, 28.82065773010254],
+      [47.03597784141986, 28.82155895233154],
+      [47.037878958764374, 28.818544149398804],
+    ],
+  },
+  {
+    name: 'Botanica',
+    polygon: [
+      [46.987505737947956, 28.88427972793579],
+      [46.98884505415609, 28.885556459426883],
+      [46.988479124097836, 28.886511325836178],
+      [46.98729349350121, 28.888131380081173],
+      [46.98627617221127, 28.886168003082272],
+    ],
+  },
+  {
+    name: 'Ciocana',
+    polygon: [
+      [47.02578377253985, 28.88381838798523],
+      [47.025578993676476, 28.887734413146973],
+      [47.029286832141274, 28.888195753097534],
+      [47.02950622279598, 28.88421535491943],
+    ],
+  },
+]
+
+function isTransportWithinTrolleyPark(transport: TransportType): boolean {
+  for (const park of trolleyParks) {
+    if (isPointInsidePolygon([transport.latitude, transport.longitude], park.polygon)) {
+      return true
+    }
+  }
+  return false
+}
+
+function isPointInsidePolygon(point: [number, number], polyPoints: number[][]): boolean {
```

Comment:
@iamandrewluca See [Determine if a point reside inside a leaflet polygon](https://stackoverflow.com/questions/31790344/determine-if-a-point-reside-inside-a-leaflet-polygon)

Reply:
Thanks, If is on stackoverflow then its true 😃